llvm/test/CodeGen/X86/atomic-scalarization.ll

```diff
@@ -0,0 +1,21 @@
+; NOTE: Assertions have been autogenerated by utils/update_llc_test_checks.py UTC_ARGS: --version 5
+; RUN: llc %s --mtriple=x86_64 -o - | FileCheck %s
+
```

Comment:
test name could be more specific, and preferably merged with an existing atomic test. With more run lines covering the different legal types.